aten/src/ATen/templates/TypeDefault.h

```diff
@@ -1,14 +1,12 @@
 #pragma once
 
-// ${generated_comment}
+#include <ATen/ATen.h>
```

Comment:
why do we still need TypeDefault?  Is it because you don't want to break extension mechanisms or you still want a place to have all the dispatch code? (i.e. done want to inline them in the methods or whatever?)

Reply:
All the Type classes are basically just acting as namespaces for now. TypeDefault is for the wrappers and registrations for non backend specific native functions.